aphrodite/modeling/layers/sampler.py

```diff
@@ -272,7 +272,8 @@ def forward(
                     sampling_tensors.output_tokens,
                     sampling_tensors.presence_penalties,
                     sampling_tensors.frequency_penalties,
-                    sampling_tensors.repetition_penalties)
+                    sampling_tensors.repetition_penalties,
+                    rep_range=rep_range)
```

Comment:
```suggestion
                    sampling_tensors.rep_range)
```

This parameter needs to be added to the sampling_metadata module.

Reply:
I'll fix on Monday!